src/bitmessageqt/foldertree.py

```diff
@@ -22,38 +29,42 @@ class AccountMixin(object):
     BROADCAST = 5
 
     def accountColor(self):
+        """"""
         if not self.isEnabled:
             return QtGui.QColor(128, 128, 128)
         elif self.type == self.CHAN:
             return QtGui.QColor(216, 119, 0)
         elif self.type in [self.MAILINGLIST, self.SUBSCRIPTION]:
-            return QtGui.QColor(137, 04, 177)
-        else:
-            return QtGui.QApplication.palette().text().color()
+            return QtGui.QColor(137, 0o4, 177)
```

Comment:
It probably should be just 4. It looks like it originated in f1b3041c9c1b835af17bc9987667b13fbab7906d but I can't find any reason for it.

Reply:
Good catch! This is autopep8 doing 'the right thing' since numbers with a leading zero in them are interpreted as octal. Reverting to just `4`...